tests/messageRoute.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { test, expect } from '@playwright/test';
```

Comment:
this import is calling the wrong module. Even though you are not using the `mount` fixture our setup is only prepared for `/expermimental-ct-react`

I suspect we should actually just `npm uninstall @playwright/test` and it's only available due to a conflict we didn't spot before

Reply:
I have updated the import statement to /expermimental-ct-react. I ran ```npm run ci``` and all tests passed before I uninstalled the @playwright/test library. When I ran it again after uninstalling it failed with ```Error: Command failed with exit code 1: npm run test```

Do we know that the experimental-ct-react library does not require the playwright/test library? Do the tests pass for you after uninstalling?

Reply:
Ok, it looks like the ct package wraps the /test package to provide the additional fixture. So probably it's when you get the message asking to run `npx install playwright` that it just reinstalls the /jest module and everything runs as normal.

My mistake here, it seems we need to keep both even though the setup files are only calling `experimental-ct-react`